### PR TITLE
Add ERWR doc

### LINKED ISSUE #2018: Docs Page "Algorithms and Methods -> On-Policy -> ERWR"

### PR DESCRIPTION
**Pull request by maliesa96**
Closes #2018

### CODE REVIEW

File: docs/user/algo_erwr.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+| **Papers**        | Using Reward-weighted Regression for Reinforcement Learning of Task Space Control :cite:`peters2007reward`                                                                 |
+|                   |                                                                                                                                                                            |
+|                   | Policy Search for Motor Primitives in Robotics :cite:`2009koberpolicy`                                                                                                     |
++-------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **Framework(s)**  | .. figure:: ./images/tf.png                                                                                                                                                |
+|                   |    :scale: 10%                                                                                                                                                             |
```
Comment (yeukfu) [inline docs/user/algo_erwr.md:10]:
```suggestion
|                   |    :scale: 20%                                                                                                                                                             |
```

File: docs/user/algo_erwr.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+|                   |    :class: no-scaled-link                                                                                                                                                  |
+|                   |                                                                                                                                                                            |
+|                   |    Tensorflow                                                                                                                                                              |
++-------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **API Reference** | `garage.tf.algos.ERWR <../_autoapi/garage/torch/algos/index.html#garage.tf.algos.erwr>`_                                                                                   |
```
Comment (yeukfu) [inline docs/user/algo_erwr.md:15]:
```suggestion
| **API Reference** | `garage.tf.algos.ERWR <../_autoapi/garage/tf/algos/index.html#garage.tf.algos.ERWR>`_                                                                                   |
```

File: docs/user/algo_erwr.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
++-------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **Examples**      | :ref:`erwr_cartpole`                                                                                                                                                       |
++-------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+Episodic Reward Weighted Regression (ERWR) is an extension of the original RWR algorithm, which uses a linear policy to solve the immediate rewards learning problem. The extension implemented here applies RWR to episodic reinforcement learning. To read more about both algorithms see the cited papers or the summary provided in this [this](https://spiral.imperial.ac.uk:8443/bitstream/10044/1/12051/7/fnt_corrected_2014-8-22.pdf) text.
```
Comment (yeukfu) [inline docs/user/algo_erwr.md:23]:
```suggestion
Episodic Reward Weighted Regression (ERWR) is an extension of the original RWR algorithm, which uses a linear policy to solve the immediate rewards learning problem. The extension implemented here applies RWR to episodic reinforcement learning. To read more about both algorithms see the cited papers or the summary provided in [this](https://spiral.imperial.ac.uk:8443/bitstream/10044/1/12051/7/fnt_corrected_2014-8-22.pdf) text.
```

**Review by yeukfu — APPROVED** [review]

**Review by krzentner — APPROVED** [review]
Thanks! Please address / accept Ruofu's changes. Otherwise, LGTM.

**Pull request merged**